libs/blocks/mobile-app-banner/mobile-app-banner.js

```diff
@@ -0,0 +1,72 @@
+import { getConfig } from '../../utils/utils.js';
+
+async function getKey(product) {
+  const config = getConfig();
+  let keyMatch = [];
+  const resp = await fetch(`${config.contentRoot ?? ''}/branch-io-key.json`);
```

Comment:
are we expecting for different keys to be used or things to change often? maybe we can set them as part of config 
 or metadata instead of performing a new call.

Reply:
Thanks for the review @narcis-radu. Yes, there'll be multiple key. One key for each product. This will be mainly managed by authors hence kept it in an excel file. They'll be authoring product name in the block. Hence if it's in an excel file they can refer to exact product name to author. 
Authors wanted to lock this file only for 1-2 GWP members per cloud to avoid any errors, hence didn't added in metadata sheet. Also this excel file will be not be that big hence didn't see any performance impact.